packages/core/test/new-api/call.ts

```diff
@@ -558,7 +558,7 @@ describe("call", () => {
       await assert.isRejected(
         validateNamedContractCall(
           future as any,
-          setupMockArtifactResolver({} as any)
+          setupMockArtifactResolver({ Another: {} as any })
```

Comment:
This is a side-effect of making `setupMockArtifactResolver` more flexible.

It now will return a generic artifact, unless your set it up with a map of contract names to artifacts.